pkg/dao/repository_configs.go

```diff
@@ -82,7 +82,7 @@ func (r *repositoryConfigDaoImpl) InitializePulpClient(ctx context.Context, orgI
 		return err
 	}
 
-	pulpClient := pulp_client.GetPulpClientWithDomain(ctx, domainName)
+	pulpClient := pulp_client.GetPulpClientWithDomain(context.Background(), domainName)
```

Comment:
I wonder if context.Todo() would make more sense here?  I think long term we may want to refactor such that the dao is created per request, so that we can get better request logging with a request id, and thus the context would need to be passed per request to the dao layer.

Reply:
I can change it to context.Todo(). Either way it was only meant to be an immediate short term fix because this is breaking things. My intention was to refactor.